Makefile

```diff
@@ -0,0 +1,44 @@
+SHELL := /bin/bash
+
+all: clean fmt docs lint test
+
+setup: ## setup development dependencies
+	curl -L https://raw.githubusercontent.com/chanzuckerberg/bff/master/download.sh | sh
+
+release: ## run a release
+	bff bump
+	git push
+
+fmt:
```

Comment:
everything but `release` is specific to cztack so I would remove that. 
```
SHELL := /bin/bash

release:
    foo
     bar
```

Reply:
How about `setup`? It seemed very general (just installing bff, right?)

Reply:
Should I reassign you after I'm done with this change?